recipes-core/images/core-image-minimal.bbappend

```diff
@@ -0,0 +1,8 @@
+copy_file_to_rootfs() {
```

Comment:
You shouldn't be bypassing Bitbake's normal method of installing items into rootfs.  These two items would normally be installed into rootfs using this bbappend but with:

```
IMAGE_INSTALL += "kernel-image kernel-devicetree"
```

The 1st would install `/boot/uImage-${KERNEL_VERSION}` and the second would install `/boot/devicetree-uImage-${MACHINE}.dtb`.

For debugging purposes, you might also find installing `/boot/vmlinux-${KERNEL_VERSION}` helpfull as well.  That can be done with `IMAGE_INSTALL += "kernel-vmlinux"`.

Reply:
@cbagwellfnc , I have told Accton this is ok for now and we FNC will make those changes.